indicators/NO_GLAC_001/R/NO_GLAC_001.qmd

```diff
@@ -0,0 +1,1731 @@
+  mutate(region = case_when(
+    id == 3 ~ "Oestlandet",
+    .default = region
+  ))
+
+```
+
+### 7.4 Dataset D - Bioclimatic regions
+
+Here I import a map dataset of bioclimatic sones and sections for Norway
+[@bakkestuen2008]. The data licence is unknown, but we have the authors
+permission to use the data (Bakkestuen, *pers. com.*). There is,
+however, a [public WMS
+service](https://kartkatalog.geonorge.no/metadata/66f09e22-23a4-43c6-a0df-426d2b317890),
+but I don't known if the data is downloadable anywhere.
+
+I combine the sones (related to heat sums and the north-south gradient)
+and sections (related to oceanity and the west-east gradient) into
+regions. There are 24 regions, but I will combine some sones and
+sections to reduce this amount. Since there are few glaciers in
+continental areas I will combine sections 4 and 5, since we don't need
+that high resolution in the EAs here. Similarly, I will combine the
+warmer sones 1 and 2. The total area of each bioclimatic region is shown
+in @fig-sonerOgSeksjoner.
+
+```{r}
+#| eval: false
+
+# file path to locally stored data
+pathD <- "R:/GeoSpatialData/BiogeographicalRegions/Norway_PCA_klima/Original/20170614_Bioklima/20170614_Bioklima.gdb"
+
+# read in the data and convert to tibbles for easy joins
+soner <- sf::read_sf(pathD, layer = "Soner2017") |> 
+  as_tibble() |>
+  mutate(Sone = case_when(
+    Sone_kode %in% c("6SO-1", "6SO-2") ~ "6SO-1-2",
+    .default = Sone_kode
+  ))
+
+# unique(soner$Sone) # where is the middle and high alpine sone?
+# I checked the WMS data and there the alpine sones are all combined
+# I check the extents against the geographical region map to ensure no big data holes
+# and it looks ok. 
+
+#soner |>
+#  mutate(area = st_area(Shape)) |>
+#  summarise(area = sum(area)/10^6) # 351 402 [km^2]
+#
+#greg |>
+#  mutate(area = st_area(geometry)) |>
+#  summarise(area = sum(area)/10^6) # 325 462.2 [km^2]
+
+seksjoner <- sf::read_sf(pathD, layer = "Seksjoner2017") |> 
+  as_tibble() |>
+  mutate(Seksjon = case_when(
+    Seksjon_ko %in% c("6SE-4", "6SE-5") ~ "6SE-4-5",
+    .default = Seksjon_ko
+  ))
+
+# unique(seksjoner$Seksjon)
+
+# join based on SSB IDs and convert to sf object again
+BCreg <- dplyr::left_join(soner , seksjoner |> select(-Shape), by = join_by(SSBID)) |>
+  mutate(BCregion = paste(Seksjon, Sone)) |>
+  sf::st_as_sf() |>
+  select(SSBID,
+  Sone,
+  Seksjon,
+  BCregion) |>
+  st_transform(st_crs(greg))
+
+```
+
+```{r makeFig}
+#| eval: false
+#| code-summary: 'Make figure'
+BCreg |>
+  ggplot(aes(x = Sone, fill = BCregion)) +
+  geom_bar() +
+  facet_grid(Seksjon ~ Sone, scales = "free") +
+  guides(fill = "none") +
+  theme(axis.text.x = element_blank(),
+    axis.title.x.bottom = element_blank()) +
+  labs(y = "km-2")
+ggsave("../img/bcreg.png")
+```
+
+```{r fig-sonerOgSeksjoner}
+#| fig-cap: "Distribution of area within each combination of bioclimatic sone and section (i.e. each bioclimatic region)."
```

Comment:
Add description of what SO1-5 and SE1-5 mean. Like which once are dry/wet etc. Or replace the facet labels with a more intuitive labels. Oceanic, strong oceanic etc.?

Reply:
Done